beacon-chain/state/cloners.go

```diff
@@ -113,7 +113,7 @@ func CopyBeaconBlockBody(body *ethpb.BeaconBlockBody) *ethpb.BeaconBlockBody {
 
 // CopyProposerSlashings copies the provided ProposerSlashing array.
 func CopyProposerSlashings(slashings []*ethpb.ProposerSlashing) []*ethpb.ProposerSlashing {
-	if slashings == nil || len(slashings) < 1 {
```

Comment:
This was causing a diff when a block had an empty array of slashings vs a nil array of slashings